File: UBB/Fourth Semester/Projecting and Programming Methods/Laboratory 06/Lab6JAVA/src/main/java/com/athleticscompetiton/domain/Round.java
package com.athleticscompetiton.domain;

public class Round {
    Long id;
    String name;

    /**
     * constructor for round
     * @param name - string, represents the name of the round
     */
    public Round(String name) {
        this.name = name;
    }

    /**
     * getter methode for id
     * @return - long, represents the round's id
     */
    public Long getId() {
        return id;
    }

    /***
     * setter methode for id
     * @param id - long, represents the round's id
     */
    public void setId(Long id) {
        this.id = id;
    }

    /***
     * getter methode for round's name
     * @return - string, represents the round's name
     */
    public String getName() {
        return name;
    }

    /**
     * setter methode for round's name
     * @param name - string, represents the round's name
     */
    public void setName(String name) {
        this.name = name;
    }
}

File: UBB/Fourth Semester/Projecting and Programming Methods/Laboratory 04/Lab4C#/AthleticsCompetition/domain/Round.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace AthleticsCompetition.domain
{
    class Round
    {
        /// <summary>
        /// constructor for round
        /// </summary>
        /// <param name="name" > string </param>
        public Round(string name)
        {
            this.Name = name;
        }

        /// <summary>
        /// getter and setter for round id
        /// </summary>
        public long Id { get; set; }

        /// <summary>
        /// getter and setter for round id
        /// </summary>
        public string Name { get; set; }

    }
}
